src/opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -92,14 +92,14 @@ PvtgTable::PvtgTable( const DeckKeyword& keyword, size_t tableIdx ) :
     }
 
 PvtoTable::PvtoTable( const DeckKeyword& keyword, size_t tableIdx) :
-    PvtxTable("P") {
+    PvtxTable("RS") {
         m_underSaturatedSchema.addColumn( ColumnSchema( "P"  , Table::STRICTLY_INCREASING , Table::DEFAULT_NONE ));
-        m_underSaturatedSchema.addColumn( ColumnSchema( "BO" , Table::RANDOM , Table::DEFAULT_LINEAR ));
+        m_underSaturatedSchema.addColumn( ColumnSchema( "BO" , Table::STRICTLY_DECREASING, Table::DEFAULT_LINEAR ));
         m_underSaturatedSchema.addColumn( ColumnSchema( "MU" , Table::RANDOM , Table::DEFAULT_LINEAR ));
 
         m_saturatedSchema.addColumn( ColumnSchema( "RS" , Table::STRICTLY_INCREASING , Table::DEFAULT_NONE ));
         m_saturatedSchema.addColumn( ColumnSchema( "P"  , Table::RANDOM , Table::DEFAULT_NONE ));
-        m_saturatedSchema.addColumn( ColumnSchema( "BO" , Table::RANDOM , Table::DEFAULT_LINEAR ));
+        m_saturatedSchema.addColumn( ColumnSchema( "BO" , Table::STRICTLY_INCREASING, Table::DEFAULT_LINEAR ));
```

Comment:
This is an older change, but I just got a real field case that fails due to this condition.  For the record, the field case runs in ECLIPSE.  As far as I can tell, there's no real requirement that the FVF be strictly increasing in the **saturated** table of `PVTO`.  There is obviously a requirement that `BO` be strictly **decreasing** for each constant value of `RS`, i.e., within each subtable of undersaturated states, but I can see no reason that the opposite must be true for the saturated table.

@GitPaean : Do you remember why you made this change?  If not, I'm tempted to reverse it to `RANDOM`.

Reply:
I do not recall. The commit comment is saying it is from the testing cases, while I do not recall which one it is. 

It is okay to me to revert it following your understanding. If we are having problem with some cases with reverting, we can discuss again. 

For your case, if possible, can you check whether Eclipse gives any complaining, warning or even actions about this?  Sometime E100 goes ahead to do the simulation even they do not like the input.

Reply:
> can you check whether Eclipse gives any complaining, warning or even actions about this?

I can, and there are are indeed a number of warnings concerning this table&mdash;i.e., warnings of the form
```
 MONOTONICITY WARNING IN PVTO TABLE NUMBER   [I]
 FVF OF SATURATED OIL (=  [BO_1]) AT RS = [RS_1] IS &lt; OR =
 FVF OF SATURATED OIL (=  [BO_2]) AT RS = [RS_2]
```
(with `BO_1 < BO_2` and `RS_1 > RS_2`).

Reply:
I do not remember clearly, but it is likely I made the change based on this type of output from E100. 

I do not have good suggestions how we should handle this.  Maybe also warning instead of exiting? 

It really comes to how much compatibility we want to have.

Reply:
> It really comes to how much compatibility we want to have.

Yeah, that's the fundamental question.  For the record, I can make this particular case run in OPM Flow if I revert `STRICTLY_INCREASING` back to `RANDOM`, but there's no telling if that will be detrimental to the overall performance of the simulator.

> Maybe also warning instead of exiting?

Agreed.  I'll see if I can think of a reasonable compromise here.